### PR TITLE
If on speaker page, meta tags are about speaker

### PR DESCRIPTION
**Pull request by hoosteeno**

### CODE REVIEW

**Comment by hoosteeno** [comment]
This PR will render https://github.com/mdn/viewsourceconf/pull/156 invalid. /cc @wilsonpage

File: layouts/includes/meta.html [hunk 1]
```diff
@@ -8,18 +8,26 @@
...
+            <meta name="twitter:image" content="https://viewsourceconf.org/assets/images/{{ share_prefix }}_twitter.jpg">
+        {% endif %}
         <meta property="og:type" content="website">
         <meta property="og:site_name" content="View Source Conference">
         <meta property="og:locale" content="en_US">
-        <meta property="og:url" content="https://viewsourceconf.org{{ site_dir }}">
-        <meta property="og:image" content="https://viewsourceconf.org/assets/images/{{ share_prefix }}_facebook.jpg">
-        <meta property="og:title" content="{{ site_title }}">
-        <meta property="og:description" content="View Source is a conference for front-end web developers. Our goal is to provide an in-depth, practical look at current and on-the-horizon technologies, with plenty of opportunities for conversation.">
+        <meta property="og:url" content="https://viewsourceconf.org/{{ path }}">
```
Comment (wilsonpage) [inline layouts/includes/meta.html]: Will this be unique now? If so, looks goooood :)

**Comment by stephaniehobson** [comment]
Much easier to read, unique meta tags also good. :+1:

File: layouts/includes/meta.html (hunk 1 above)
Comment (hoosteeno) [inline layouts/includes/meta.html]: Yes!

**Pull request merged**